app/controllers/editorial/submissions_controller.rb

```diff
@@ -10,7 +10,7 @@ class Editorial::SubmissionsController < Editorial::SectionsController
 
 
     def index
-      @submissions = scope.open.for(current_user)
+      @submissions = scope.open
```

Comment:
I think you mean 'there isn't a much an **author** can do with any but their own' which is a fair point. 

I can make the controller a bit smarter so it returns only actionable submissions in the context of the currently logged in user's role for the current section. 

BTW, `for` scope did indeed have a hidden order inside it... I'm going to shift that out into its own scope.